src/components/Tournament/TournamentIcon.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import {
+  Glyphicon
+} from 'react-bootstrap';
+
+const TournamentIcon = ({icon}) => {
+  return (
+    <section className="tournament-icon">
```

Comment:
Why does 'section' element have wrapper role? By semantic it can be ...

Reply:
Of course because of semantic, so what should we use here? I'm open to suggestions =)

Reply:
It might be just obvious DIV. But section it's a independent logical block)

Reply:
DIV or article?)